ui/Dockerfile

```diff
@@ -3,8 +3,11 @@ ARG NGINX_IMAGE_VERSION=1.15.8
 FROM node AS build-step
 
 WORKDIR /home/node
-COPY . /home/node/
+COPY package*.json /home/node/
```

Comment:
I think we do not have to update since package*.json is part of '/ui'.

Reply:
Yeah, but currently the buildchain will re-trigger the build if any file under `/ui` is modified, whereas with you change we only care about the `package*.json`.

If you don't update then you'll trigger useless builds (that will be fast thanks to better caching with this change), so it's better to update to avoid it :)

BTW, shouldn't your check complains about this mismatch @jbertran ?